.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Pull Request template
```

Comment:
Can we use the format mentioned here for the pull request template with few additional questions and checklists. 
https://github.com/openebs/maya/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Thanks @kmova, updated the changes.